### PR TITLE
Override

### PR DESCRIPTION
**Pull request by ducks23**

### CODE REVIEW

File: slurm_ops_manager/slurm_snap_manager.py [hunk 1]
```diff
@@ -123,7 +123,7 @@ def upgrade(self):
         # instead).
         self.setup_system()
 
-    def _provision_snap_systemd_service_override_file(self):
+    def write_systemd(self):
```
Comment (jamesbeedy) [inline slurm_ops_manager/slurm_snap_manager.py:126]: I would like to see a more descriptive function name here.... I actually liked your first go, how about `write_systemd_service_override_file()`?

Comment (jamesbeedy) [reply slurm_ops_manager/slurm_snap_manager.py:126]: this function should go in slurm_ops_base and use `self._slurm_systemd_service` to template the override.conf location.

Comment (jamesbeedy) [reply slurm_ops_manager/slurm_snap_manager.py:126]: add a doc string to this function

**Review by jamesbeedy — CHANGES_REQUESTED** [review]
change the function name to something more descriptive, add docstring, move to slurm_ops_base, use self._slurm_systemd_service.

File: slurm_ops_manager/slurm_snap_manager.py (hunk 1 above)
Comment (ducks23) [reply slurm_ops_manager/slurm_snap_manager.py:126]: I added a doc string to make function more descriptive

File: slurm_ops_manager/slurm_snap_manager.py [hunk 2]
```diff
@@ -123,7 +123,9 @@ def upgrade(self):
         # instead).
         self.setup_system()
 
-    def _provision_snap_systemd_service_override_file(self):
+    def write_systemd_file(self):
```
Comment (jamesbeedy) [inline slurm_ops_manager/slurm_snap_manager.py:126]: move this function needs to `slurm_ops_base.py`, rename it to be `write_slurm_process_systemd_override_conf()`

**Review by jamesbeedy — CHANGES_REQUESTED** [review]
fix

**Pull request closed without merging**